zapisy/apps/notifications/signals.py

```diff
@@ -177,6 +177,25 @@ def notify_board_members_about_voting(sender: Thesis, **kwargs) -> None:
         }, target))
 
 
+@receiver(event_decision, sender=None)
+def notify_event_creator_about_decision(sender: None, **kwargs) -> None:
```

Comment:
```suggestion
def notify_event_author_about_decision(sender: None, **kwargs) -> None:
```
Spójnie z nazwą pola modelu.